packages/nimble-components/src/radio-group/styles.ts

```diff
@@ -23,6 +27,12 @@ export const styles = css`
         flex-direction: row;
     }
 
+    .label-container {
+        display: flex;
+        gap: ${smallPadding};
+        margin-bottom: ${smallPadding};
```

Comment:
Looks like this introduces some spacing even when there is no label. I'm not sure how much we care about the no-label use case, but would it be better to apply the `margin-bottom` to `::slotted([slot="label"]), .error-icon`  instead?
![image](https://github.com/user-attachments/assets/33aa9ab5-da45-43ef-b5a4-2f5bc9045462)

Reply:
I debated this myself. The thing I didn't want to have happen, though, is for the control with no label to shift when `error-visible` was toggled, which is why I decide to leave the space there all the time.

I'll leave the conversation open for others to weigh in.

Reply:
I actually realized that as-is the control still shifts when the error icon comes & goes without a label. I'll fix that, but it'll ultimately mean more spaces is reserved even when there isn't an error or a label.

Reply:
+1 for avoiding shifts when transitioning in and out of error states